tests/tailor.js

```diff
@@ -249,7 +249,6 @@ describe('Tailor', () => {
                 .then(response => {
                     assert.equal(response.statusCode, 200);
                     assert.deepEqual(response.headers['set-cookie'], [cookie]);
-                    assert.equal(response.headers.link, 'http://link');
```

Comment:
Disallow primary fragments from overriding link headers since primary fragments have no context of module loading and piping mechanism.

Reply:
We are also including their assets after the loader asset.